lighthouse-core/gather/gather-runner.js

```diff
@@ -70,8 +70,16 @@ class GatherRunner {
   static loadBlank(driver, url) {
     // The real about:blank doesn't fire onload and is full of mysteries (https://goo.gl/mdQkYr)
     // To improve speed and avoid anomalies (https://goo.gl/Aho2R9), we use a basic data uri page
-    url = url || `data:text/html,${logoPageSource}`;
-    const blankPageUrl = `data:text/html,${blankPageSource}`;
+
+    let blankPageUrl;
+
+    if (typeof chrome !== 'undefined') {
```

Comment:
I don't mind this as a quickfix but could we maybe move this up to lighthouse options so we don't have to do a dirty check inside our gatherer for if we are a chrome extension

Reply:
This PR is more of a proof of concept than a final solution 👍 It definitely needs a clean up